great_expectations/datasource/data_connector/configured_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,140 @@
+import logging
+from typing import Dict, List, Optional, cast
+
+from great_expectations.core.batch import BatchDefinition, BatchSpec
+from great_expectations.core.batch_spec import GlueCatalogBatchSpec
+from great_expectations.datasource.data_connector.configured_asset_sql_data_connector import (
+    ConfiguredAssetSqlDataConnector,
+)
+from great_expectations.execution_engine import ExecutionEngine, SparkDFExecutionEngine
+
+try:
+    import boto3
+except ImportError:
+    boto3 = None
+
+logger = logging.getLogger(__name__)
+
+
+class ConfiguredAssetGlueCatalogDataConnector(ConfiguredAssetSqlDataConnector):
+    """
+    Extension of ConfiguredAssetSqlDataConnector used to connect to AWS Glue Data Catalog
+
+    The ConfiguredAssetGlueCatalogDataConnector is one of two classes (InferredAssetGlueCatalogDataConnector
+    being the other one) designed for connecting to data through AWS Glue Data Catalog.
+
+    A ConfiguredAssetGlueCatalogDataConnector requires an explicit listing of each DataAsset you want to
```

Comment:
@lccasagrande If not a big problem, could we please the generic language of "one" instead of "you" in docstrings.  Thank you!

Reply:
Sure, fixed